src/main/java/core/basesyntax/operations/BalanceHandler.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.operations;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.model.FruitTransaction;
+
+public class BalanceHandler implements OperationHandler {
+    private final StorageDao storageDao = new StorageDaoImpl();
+
+    @Override
+    public Integer getHandler(FruitTransaction fruitTransaction) {
+        if (fruitTransaction == null) {
+            throw new RuntimeException("Fruit transaction is null "
+                    + fruitTransaction);
+        }
+        if (!fruitTransaction.getOperation().equals(FruitTransaction.Operation.BALANCE)) {
+            throw new RuntimeException("Can' find balance "
+                    + fruitTransaction);
+        }
```

Comment:
```suggestion
        if (fruitTransaction == null) {
            throw new IllegalArgumentException("Fruit transaction is null");
        }
        if (!FruitTransaction.Operation.BALANCE.equals(fruitTransaction.getOperation())) {
            throw new RuntimeException("Balance handler can't handle an unsuitable transaction operation type: " + fruitTransaction.getOperation());
        }
```

Reply:
make validation consistent across handlers, if you check for null and suitability of operation type - check everywhere